torch/csrc/jit/codegen/cuda/lower_shift.cpp

```diff
@@ -223,7 +228,6 @@ void HaloInfo::build(Fusion* fusion) {
   }
 
   if (isDebugDumpEnabled(DebugDumpOption::Halo)) {
-    std::cout << toString() << std::endl;
```

Comment:
Is this just accidentally removed? Or is there any problem with this? I often use this debug option when working on shift/gather.

Reply:
accident, reverting...